### DISCUSSION TITLE
External access to Valetudo is blocked

### DISCUSSION THREAD

**Discussion by stullko** [Hypfer/Valetudo#2027] [Q&A / Support] [answered]
### Are you already using Valetudo?

Yes, for a few months

### Your questions

I have been using Valetudo for a long time, I needed to change the router's IP address and DHCP ranges, the wifi SSID and password have changed, the Roborock 5 connects to wifi, I see it on the network, but the UI does not show up for me, only this error: External access to Valetudo is blocked.
Valetudo has the correct new address, but can't access UI.

**Comment by Hypfer** [comment]
Make sure that you're using a DHCP range that is part of these: https://en.wikipedia.org/wiki/Private_network#Private_IPv4_addresses

**Comment by stullko** [comment]
I changed only the ranges from 10.10.10.xx to 11.11.11.xx.

**Reply by Hypfer** [reply, marked as answer]
`11.11.11.x` is not part of the list of valid private IPv4 ranges that I've linked you.
